wom/src/main/scala/wom/graph/CallNode.scala

```diff
@@ -51,6 +51,7 @@ object TaskCall {
     
     // Creates an identifier for an input or an output
     // The localName is the name of the input or output
+    
```

Comment:
While you're here, can you covert this comment block from `//` to `/* */`? I would say `/** */`, but I don't think local-def may use `/** */`.